src/main/java/io/debezium/connector/yugabytedb/connection/OpId.java

```diff
@@ -74,7 +74,7 @@ public static OpId valueOf(String stringId) {
      * </pre>
      */
     public String toSerString() {
-        String keyStr = Base64.getEncoder().encodeToString(key);
+        String keyStr = (key == null) ? "" : Base64.getEncoder().encodeToString(key);
```

Comment:
Added the distinction in the `toString()` method. `toSerString()` is only used to store checkpoint values as offsets.

Reply:
This distinction between NULL value (semantics being NOT PRESENT) vs empty string (PRESENT and empty string value) is important in all contexts because these two represent different things. I think, in this context of checkpoints, it is all the more important to capture this distinction

Reply:
Added the relavant distinction in the methods. `null` values will now be represented by a string "null" and empty values with empty string.